Classes/Decoder/UrlDecoder.php

```diff
@@ -400,9 +400,11 @@ protected function decodeFixedPostVars($pageId, array &$pathSegments) {
 
 			$previousValue = '';
 			foreach ($postVars as $postVarConfiguration) {
+				if (!is_array($postVarConfiguration)) {
```

Comment:
This should not be needed because there is a call to `getConfigurationForPostVars`, which resolved postVars to arrays.